test/scaladoc/scalacheck/IndexScriptTest.scala

```diff
@@ -46,7 +46,8 @@ object Test extends Properties("IndexScript") {
           "scala.tools.nsc",
           "scala.tools.nsc.doc",
           "scala.tools.nsc.doc.html",
-          "scala.tools.nsc.doc.html.page"
+          "scala.tools.nsc.doc.html.page",
+          "scala.tools.nsc.interpreter"     // incurred by Contexts.isRootImport
```

Comment:
@retronym I haven't nailed this down, but cracking open the test was enough trauma for one evening.